### PR TITLE
created a preprocessor Median blur

### PR DESCRIPTION
**Pull request by rudrapsc**
solved the alignment issue in template.json file of sample 6(OMRChecker\samples\sample6\template.json) along with using median blur as a medium for smoothing .
created a new preprocessor called MedianBlur (OMRChecker\src\processors\builtins.py).

### CODE REVIEW

File: src/processors/builtins.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import numpy as np
 
 from .interfaces.ImagePreprocessor import ImagePreprocessor
-
+from src.logger import logger
```
Comment (Udayraj123) [inline src/processors/builtins.py:5]: remove this line

File: src/processors/builtins.py [hunk 2]
```diff
@@ -13,7 +13,6 @@ def output_level(value, low, high, gamma):
                 return 255
             inv_gamma = 1.0 / gamma
             return (((value - low) / (high - low)) ** inv_gamma) * 255
-
```
Comment (Udayraj123) [inline src/processors/builtins.py:16]: keep newline as it is. Avoid whitespace changes

**Comment by rudrapsc** [comment]
this is the output before editing the code

Screenshot_20221217_031619




this is the output after editing the code

Screenshot_20221217_031705

**Comment by rudrapsc** [comment]
I added a plugin class MedianBlur(OMRChecker\src\processors\builtins.py), this plugin uses median blur instead of gaussian blur 
median blur is an algorithm based library.
Here, the central element of the image is replaced by the median of all the pixels in the kernel area.
due to this ,the darkened circles appears more clearly and becomes easy to read.

**Comment by Udayraj123** [comment]
Looks good! Thanks

**Review by Udayraj123 — APPROVED** [review]

**Pull request merged**